examples/use_toolkit.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
```

Comment:
can we change this example to something like `use_static_tools.py`, the current name is quite confusing.

Reply:
or maybe we can just combine this example with the existing `use_tools.py`, simply by changing get_tools() to get_tools_async()?